builder/Dockerfile

```diff
@@ -1,17 +1,33 @@
-FROM golang AS builder
-RUN mkdir /app
-ADD . /app
-WORKDIR /app
-RUN CGO_ENABLED=0 GOOS=linux go build github.com/Azure/aks-periscope/cmd/aks-periscope
+# Builder
+FROM golang:alpine AS builder
 
+ENV GO111MODULE=on \
+    CGO_ENABLED=0 \
+    GOOS=linux \
+    GOARCH=amd64
+
+WORKDIR /build
+
+COPY go.mod .
+COPY go.sum .
+RUN go mod download
+
+COPY . .
+
+RUN go build ./cmd/aks-periscope
+
+# Runner
 FROM alpine
+
 RUN apk --no-cache add ca-certificates curl openssl bash
+
 ADD https://storage.googleapis.com/kubernetes-release/release/v1.16.0/bin/linux/amd64/kubectl /usr/local/bin/kubectl
 RUN chmod +x /usr/local/bin/kubectl
+
 RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 \
     && chmod +x get_helm.sh \
     && ./get_helm.sh
-RUN mkdir /app
-WORKDIR /app
-COPY --from=builder /app/aks-periscope .
-CMD ["./aks-periscope"]
+
+COPY --from=builder /build/aks-periscope /
```

Comment:
🐛 I think `.` lets the bash to identify and run, or might be the `entry point` below needs a mechanism to tell `/` to run with bash. 

Working image with below change: `tatsat/docker-changes-test`

**Fix**

```
COPY --from=builder /build/aks-periscope .

ENTRYPOINT ["./aks-periscope"]
```


<img width="521" alt="Screen Shot 2021-07-02 at 12 01 09 PM" src="https://user-images.githubusercontent.com/6233295/124202001-8f5a4080-db2d-11eb-9c0c-d48c7d6e5dbb.png">